VERSION.cmake

```diff
@@ -1,6 +1,6 @@
 set (DEFAULT_LIBDNF_MAJOR_VERSION 0)
 set (DEFAULT_LIBDNF_MINOR_VERSION 51)
-set (DEFAULT_LIBDNF_MICRO_VERSION 0)
+set (DEFAULT_LIBDNF_MICRO_VERSION 1)
```

Comment:
Again,I'd prefer to bump `MINOR` here, not `MICRO`, this is much more than a bugfix.

Reply:
Fixed.